### PR TITLE
excluding select2 docs from the package

### PR DESCRIPTION
**Pull request by Eraldo**
It caused missing relative paths.

### CODE REVIEW

File: MANIFEST.in [hunk 1]
```diff
@@ -9,3 +9,5 @@ include src/dal_select2/static/autocomplete_light/vendor/select2/dist/css/select
 include src/dal/static/autocomplete_light/jquery.init.js
 include src/dal/static/autocomplete_light/autocomplete.init.js
 include src/dal/static/admin/js/jquery.init.js
+recursive-include src/dal_select2/static/autocomplete_light/vendor/select2
+prune src/dal_select2/static/autocomplete_light/vendor/select2/docs
```
Comment (jpic) [inline MANIFEST.in]: Are you sure we want the prune line here, since we need the fonts ?

File: MANIFEST.in [hunk 2]
```diff
@@ -9,3 +9,5 @@ include src/dal_select2/static/autocomplete_light/vendor/select2/dist/css/select
 include src/dal/static/autocomplete_light/jquery.init.js
 include src/dal/static/autocomplete_light/autocomplete.init.js
 include src/dal/static/admin/js/jquery.init.js
+recursive-include src/dal_select2/static/autocomplete_light/vendor/select2
```
Comment (jpic) [inline MANIFEST.in]: Seems like this makes lines 7 and 8 unnecessary, in this case could we perhaps remove them ?

File: MANIFEST.in (hunk 1 above)
Comment (jpic) [inline MANIFEST.in]: Ok, fonts are not in the docs, my bad !

File: MANIFEST.in (hunk 2 above)
Comment (jpic) [inline MANIFEST.in]: Also, don't you need to add the fonts there ?

Comment (Eraldo) [inline MANIFEST.in]: I agree.

Comment (Eraldo) [inline MANIFEST.in]: I thought that it will include everything if I do not specify a pattern for the recursive-include command.

Comment (jpic) [inline MANIFEST.in]: Oh, didn't know that, pretty cool !

**Comment by jpic** [comment]
Merged locally, released in 3.0.4.

Thanks !!!

**Pull request closed without merging**